exads/doc/README.md

```diff
@@ -0,0 +1,46 @@
+# exadsBidAdapter
```

Comment:
Sure Pierre.

As mentioned yesterday by @Sofiabarreira, she has already planned to write the documentation to put into Confluence after QA testing.

I'll give her all information and support for doing that.

Reply:
@giuseppe-exads @Sofiabarreira OK, just remember to remove this content from the PR before merging it. It could be temporarily placed somewhere else if needed!

Reply:
You are right @broucz. I'll put it in another place before the PR.